programs/local/LocalServer.cpp

```diff
@@ -727,7 +727,6 @@ void LocalServer::printHelpMessage([[maybe_unused]] const OptionsDescription & o
 void LocalServer::addOptions(OptionsDescription & options_description)
 {
     options_description.main_description->add_options()
-        ("database,d", po::value<std::string>(), "database")
```

Comment:
Removing it from `LocalServer.cpp` because it is a duplicate as it is already present here https://github.com/ClickHouse/ClickHouse/blob/ebdf4d2702bcf773b01aec59780211d207073727/src/Client/ClientBase.cpp#L1678
which are common options for client and local:
https://github.com/ClickHouse/ClickHouse/blob/ebdf4d2702bcf773b01aec59780211d207073727/src/Client/ClientBase.cpp#L1669